src/core/js/scriptLoader.js

```diff
@@ -30,7 +30,8 @@
                 a11y: 'libraries/jquery.a11y',
                 scrollTo: 'libraries/scrollTo',
                 bowser: 'libraries/bowser',
-                'enum': 'libraries/enum'
+                enum: 'libraries/enum',
+                jqueryMobile: 'libraries/jquery.mobile.custom'
```

Comment:
I meant in general, it's a framework after all and we're not the only people who might want to use the swipe action.